apps/judicial-system/backend/src/app/modules/case/dto/internalCreateCase.dto.ts

```diff
@@ -11,9 +17,11 @@ export class InternalCreateCaseDto {
   readonly type!: CaseType
 
   @IsNotEmpty()
-  @IsString()
+  @IsArray()
+  @ArrayMinSize(1)
+  @IsString({ each: true })
   @ApiProperty()
-  readonly policeCaseNumber!: string
+  readonly policeCaseNumbers!: string[]
```

Comment:
This should probably not be changed as this is called form the xrd-api which in turn is called from LÖKE. We must not break our contract with LÖKE.

Reply:
Fixed in `xrd-api` see https://github.com/island-is/island.is/pull/8248/commits/ccbd32566913b4755f7fcb377ec9a3cec5f876f5